.github/workflows/nv-nightly.yml

```diff
@@ -25,7 +28,7 @@ jobs:
 
       - name: Install pytorch
         run: |
-          pip install -U --cache-dir $TORCH_CACHE torch==1.13.1 torchvision --index-url https://download.pytorch.org/whl/cu117
+          pip install -U --cache-dir $TORCH_CACHE torch torchvision --index-url https://download.pytorch.org/whl/cu118
```

Comment:
Switch to nightly feed

Reply:
Nevermind, that's not the original meaning of nightly in this, these are nightly tests, not to keep up with torch nightly releases.